storage/mysql/log_storage_test.go

```diff
@@ -734,11 +752,25 @@ func TestGetTreeRevisionAtSize(t *testing.T) {
 		}
 
 		// But an intermediate value shouldn't work
-		treeRevision3, err := tx.GetTreeRevisionAtSize(21)
+		treeRevision3, err := tx.GetTreeRevisionAtSize(21, true)
 
 		if err == nil {
```

Comment:
Yes this is some of the oldest code in the repo and needs to be looked at again but I was trying to limit the scope for now.